src/components/core/InteractiveLink/InteractiveLink.tsx

```diff
@@ -81,17 +95,59 @@ export default function InteractiveLink({
   return null;
 }
 
-export const StyledAnchor = styled.a<{ disabled?: boolean }>`
-  color: ${({ disabled }) => (disabled ? colors.porcelain : colors.shadow)};
+export function InteractiveLinkNeedsVerification({
+  inheritLinkStyling,
+  href,
+  children,
+}: {
+  inheritLinkStyling: boolean;
+  href: string;
+  children: ReactNode;
+}) {
+  const { showModal } = useModalActions();
+  const handleClick = useCallback(
+    (href) => {
+      // track('What to track here?');
```

Comment:
great Q, let's just do this!

```js
      track('Link Click', {
        to: href,
        needsVerification: true
      });
```

and in the above tracking we can use `needsVerification: fakse`